test/verify/check-system-info

```diff
@@ -559,33 +507,49 @@ revision\t: 2.3 (pvr 004e 1203)
         b.enter_page('/system/hwinfo')
         b.wait_in_text('#hwinfo-system-info-list .hwinfo-system-info-list-item:nth-of-type(2) .pf-c-description-list__group:nth-of-type(1) dd', "2x POWER9 (architected), altivec supported")
 
-        # Memory details should be shown for generic QEMU memory
-        # demidecode not available on certain images
+        # Memory details should be shown from our mocked DMI information from systemd's test files.
         if not m.ostree_image:
             b.wait_in_text('#hwinfo #memory-listing' + heading_selector, "Memory")
             b.wait_in_text('#hwinfo #memory-listing table', "DIMM")
             b.wait_in_text('#hwinfo #memory-listing table', "RAM")
 
-            # Test more specific memory data with a fake dmidecode
-            m.write('/tmp/dmidecode', dmidecode)
-            m.execute('chmod +x /tmp/dmidecode; mount -o bind /tmp/dmidecode $(which dmidecode)')
-            self.addCleanup(m.execute, 'until umount $(which dmidecode); do sleep 1; done')
+            m.upload(["verify/files/dmi/smbios_entry_point", "verify/files/dmi/Lenovo-ThinkPad-X280.bin"], "/tmp/")
+            m.execute("mount -o bind /tmp/smbios_entry_point /sys/firmware/dmi/tables/smbios_entry_point")
```

Comment:
How about mounting /sys/firmware/dmi/tables/ directory, let m.upload() copy directly into that, and unmount it just once at cleanup?